includes/payment-methods/class-wc-stripe-upe-payment-method-blik.php

```diff
@@ -53,10 +53,16 @@ public function get_retrievable_type() {
 	/**
 	 * Returns testing instructions to be printed at checkout in test mode.
 	 *
-	 * @param bool $show_optimized_checkout_instruction Whether this is being called through the Optimized Checkout instructions method. Used to avoid an infinite loop call.
 	 * @return string
 	 */
 	public function get_testing_instructions( $show_optimized_checkout_instruction = false ) {
+		if ( $show_optimized_checkout_instruction ) {
```

Comment:
To be explicit, should we check for any non-false value?
```suggestion
		if ( false !== $show_optimized_checkout_instruction ) {
```